Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -389,7 +398,7 @@ def setup_(self, step, stepName, stepDict, k, properties):
             stepDict[stepName][k] = None
         elif 'Reco' in step or 'HARVEST' in step: stepDict[stepName][k] = merge([{'--procModifiers': 'seedingDeepCore'}, stepDict[step][k]])
     def condition(self, fragment, stepList, key, hasHarvest):
-        result = (fragment=="QCD_Pt_1800_2400_14" or fragment=="TTbar_14TeV" ) and ('2021' in key or '2024' in key) and hasHarvest
+        result = (fragment=="QCD_Pt_1800_2400_14" or fragment=="TTbar_14TeV" ) and ('2021' in key or '2024' in key) and ('FS' not in key) and hasHarvest
```

Comment:
This is applied here to the class `UpgradeWorkflow_seedingDeepCore`, while in master it was applied to `UpgradeWorkflow_trackingOnlyRun3CkfPixelLessStep` (and not to `UpgradeWorkflow_seedingDeepCore`): please check and comment.

Reply:
Thx, fixed.
I make everything to be the same as master for now. The cleanup of unused workflow will be done in master-only after. There is no need to do additional clean up in 12_4.